app/views/ReportView/components/CancerPredisposition/columnDefs.ts

```diff
@@ -39,15 +39,11 @@ const columnDefs: ColDef[] = [{
       return `${variant.gene.name} ${variant.cnvState}`;
     }
     if (variantType === 'sv') {
-      return `(${
-        variant.gene1.name || '?'
-      },${
-        variant.gene2.name || '?'
-      }):fusion(e.${
-        variant.exon1 || '?'
-      },e.${
-        variant.exon2 || '?'
-      })`;
+      return `(${variant.gene1.name || '?'
```

Comment:
this change seems to be style only and doesn't seem related to the PR or the commit message. was this just autofixed or something like that?

Reply:
yeah autofixed